content/en/blog/_posts/2023-08-15-pkgs-k8s-io-introduction.md

```diff
@@ -18,17 +18,33 @@ This blog post contains information about these new package repositories,
 what does it mean to you as an end user, and how to migrate to the new
 repositories.
 
+**Update (August 31, 2023): the legacy Google-hosted repositories are deprecated
+and will be frozen starting with September 13, 2023.
+Check out [the deprecation announcement](/blog/2023/08/31/legacy-package-repository-deprecation/)
+for more details about this change.**
+
 ## What you need to know about the new package repositories?
```

Comment:
When I've updated blogs in the past, I've noted that I've updated that line or section.
Do we need to do that here?
```suggestion
## What you need to know about the new package repositories? (Updated on August 31, 2023)
```

Reply:
I like this idea!